cmd/prysmctl/testnet/generate_genesis.go

```diff
@@ -0,0 +1,246 @@
+package testnet
+
+import (
+	"context"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	fastssz "github.com/prysmaticlabs/fastssz"
+	"github.com/prysmaticlabs/prysm/v3/cmd"
+	"github.com/prysmaticlabs/prysm/v3/config/params"
+	"github.com/prysmaticlabs/prysm/v3/io/file"
+	ethpb "github.com/prysmaticlabs/prysm/v3/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v3/runtime/interop"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var generateGenesisStateCmd = &cli.Command{
+	Name:   "generate-genesis",
+	Usage:  "Generate a beacon chain genesis state from specified configs",
```

Comment:
I'd get rid of "from specified configs" - unclear what "specified configs" means, probably clear enough without it.